### PR TITLE
Document anonymous universes (PR #544).

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: doc/refman/Universes.tex [hunk 1]
```diff
@@ -332,8 +332,18 @@ \subsection{Polymorphic definitions}
...
+minimized. Flexible anonymous universes can be produced with an
+underscore or by omitting the annotation to a polymorphic definition.
+
+\begin{coq_example}
+  Check (fun x -> x) : Type -> Type.
+  Check (fun x -> x) : Type -> Type@{_}.
+
+  Check le@{k _}.
+  Check le.
+\end{coq_example}
```
Comment (SkySkimmer) [inline doc/refman/Universes.tex]: I hope this behaviour will be stable otherwise this part of the doc will become nonsensical without telling anyone.

**Comment by SkySkimmer** [comment]
I wanted to get gitlab to make documentation artifacts but then I looked at it and everything is broken. I didn't notice because fiat-parsers trained me to ignore gitlab failures while #849 and #782 waited.

**Comment by Zimmi48** [comment]
Next time it happens we should block Travis tweaks if they are not similarly implemented for GitLab (unless they are harmless).

Can you check that my addition to the CHANGES file regarding anonymous universes is fine? (#972)

**Comment by SkySkimmer** [comment]
It might be worth mentioning that they can be used for instances. I don't know how exhaustive CHANGES are supposed to be though.

**Comment by SkySkimmer** [comment]
Oops I used -> instead of => https://gitlab.com/SkySkimmer/coq/-/jobs/29530734/artifacts/raw/install/share/doc/coq/html/refman/Reference-Manual032.html#sec881

**Comment by SkySkimmer** [comment]
Good to go.
https://gitlab.com/SkySkimmer/coq/-/jobs/29540947/artifacts/raw/install/share/doc/coq/html/refman/Reference-Manual032.html#sec881

**Pull request merged**